Source/WebCore/Modules/encryptedmedia/MediaKeySession.cpp

```diff
@@ -241,12 +234,11 @@ void MediaKeySession::load(const String& sessionId, Ref<DeferredPromise>&& promi
 
     // 1. If this object is closed, return a promise rejected with an InvalidStateError.
     // 2. If this object's uninitialized value is false, return a promise rejected with an InvalidStateError.
-# if 0 /* Commenting this condition check for loading a closed persistent session */
```

Comment:
You need to rebase these patches in a coherent sequence. This was added earlier and removed now. Same with the below.